app/controllers/settings.js

```diff
@@ -32,6 +32,15 @@ export default Controller.extend({
     return isPrivate && settings.hasOwnProperty('allow_config_imports');
   }),
 
+  showBetaFeatures: computed('showConfigValidationSwitches', function () {
+    return this.get('showConfigValidationSwitches');
+  }),
```

Comment:
This could possibly be simplified by using `reads` rather than computed. It would be like:
```
import { alias, filterBy, reads } from '@ember/object/computed';

// Some other controller stuff here

  showBetaFeatures: reads('showConfigValidationSwitches'),
```

Reply:
done 🎉